Framework/CurveFitting/src/Algorithms/NormaliseByPeakArea.cpp

```diff
@@ -314,10 +308,10 @@ void NormaliseByPeakArea::saveToOutput(const API::MatrixWorkspace_sptr &accumWS,
 
   if (m_sumResults) {
     const size_t npts(accumWS->blocksize());
-    auto &accumY = accumWS->dataY(0);
-    auto &accumE = accumWS->dataE(0);
-    const auto accumYCopy = accumWS->readY(0);
-    const auto accumECopy = accumWS->readE(0);
```

Comment:
Why is this making a copy of the vectors? Do you see a reason that I am overlooking?